app/views/groups/index.html.erb

```diff
@@ -40,7 +40,7 @@
     <% end %>
   </div>
   
-  <div class="row mt-4 text-center justify-content-center">
-    <%= link_to "Add Group", new_group_path(), class: "btn btn-primary col-md-4" %>
+  <div class="container bottom-element">
+    <%= link_to "Add Group", new_group_path(), class: "btn btn-primary w-100" %>
```

Comment:
- [ Required ] I think it would be nice if you implement the previous reviewer's suggestion in the exact manner he described, the `Add Group` button should have the `greenish` color and `whitish` border as illustrated on the Behance design. Currently, your design has a `bluish` button without a `whitish` border around it. Please see the screenshot below and rectify this.

## App Screenshot vs Design screenshot
| Your design | Behance design |
|-----------------|------------------------|
| ![Screenshot from 2023-11-26 02-10-00](https://github.com/chudisoft/budget_manager/assets/16392046/27be9066-5032-4f2d-9235-11da158ec2c5) | ![Screenshot from 2023-11-26 02-12-48](https://github.com/chudisoft/budget_manager/assets/16392046/8e09139d-bf4b-4cb9-86a0-8c4e8a1ea599) |